src/components/ProjectMainContent.js

```diff
@@ -10,13 +12,22 @@ const components = {
 };
 
 const ProjectMainContent = (props) => {
-  const { mdx, project } = props;
+  const [JsxMdx, setJsxMdx] = useState(null);
```

Comment:
`nit`: curious why the state is capitalized, is that since we're making it a component?

Reply:
i forgot about this! but yes, because it's being used as a component but i'm not married to it either way